.gitignore

```diff
@@ -12,3 +12,4 @@
 
 # Ignore Gems
 *.gem
+.idea/
```

Comment:
I think this is not relevant, is it?

Reply:
Good point! Something I set up for my local dev environment and forgot to remove!